### DISCUSSION TITLE
date field

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#16647] [General] [answered]
Hi, can I restrict the date field to only select a future date?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Look at this:
![image](https://user-images.githubusercontent.com/10198654/201594088-3259edd7-cd17-4123-8cea-fbe5587031b7.png)

Or see this validator:
https://github.com/YetiForceCompany/YetiForceCRM/blob/4def22a6a3850a7e2cd03800ce969c409d1d595b/modules/Vtiger/models/Field.php#L1217-L1219
Example how to use:
https://github.com/YetiForceCompany/YetiForceCRM/blob/4def22a6a3850a7e2cd03800ce969c409d1d595b/modules/ActivityRegister/models/Field.php#L17-L28
